tornjak-frontend/src/redux/reducers/tornjakReducer.ts

```diff
@@ -1,14 +1,17 @@
 import {
     GLOBAL_MESSAGE,
     GLOBAL_CLICKED_DASHBOARD_TABLE,
+    TornjakReducerStateType,
+    TornjakMessageAction,
+    ClickedDashboardTableAction,
 } from '../actions/types';
 
-const initialState = {
+const initialState: TornjakReducerStateType = {
     globalErrorMessage: "",
     globalClickedDashboardTable: "",
 };
 
-export default function tornjakReducer(state = initialState, action) {
+export default function tornjakReducer(state: TornjakReducerStateType = initialState, action: TornjakMessageAction | ClickedDashboardTableAction) {
```

Comment:
taking in one or two types in  a function seems a little confusing. If there are two different usecases, I feel like having two functions with a common helper may make it easier to intepret (as well as augmenting the function name to describe the context of the function's use)

Reply:
That's one way of doing it, but this Tornjak reducer hosts all the reducers for Tornjak-specific actions and this is an easier way of implementing action types. I had it that way before but it gets bulky with multiple redundant functions.  (to remove the confusion I removed the two action inputs into the function and bundled them into one action in redux types like the other reducers.